client.go

```diff
@@ -246,6 +281,16 @@ func (s *Client) Run(ctx context.Context,
 	mainCtx, mainCancel := context.WithCancel(ctx)
 	defer mainCancel()
 
+	// Start our liquidity manager. This loop will exit when our main ctx
+	// is cancelled.
+	s.wg.Add(1)
+	go func() {
+		if err := s.manager.Run(mainCtx); err != nil {
```

Comment:
Isn't it the case that with just suggestions, that the liquidity manager only needs to react to rpc calls? I am wondering why it needs its own thread. Or is that only to synchronize the config? If we move to persistent storage of that, the sync would be handled by the db.

Reply:
Just to synchronize cfg, reduced this to a lock for now.